modules/rosapps/applications/net/roshttpd/httpd.cpp

```diff
@@ -461,22 +461,22 @@ VOID CHttpDaemonThread::Execute()
 		            case HTTPD_START: {
 			            if (Daemon->GetState() == hsStopped)
 				            Daemon->Start();
-					    break;
+				    break;
```

Comment:
Broken indentation (mix of tabs and spaces).

Reply:
Kinda fixed, but this file is fucked up completely)